### DISCUSSION TITLE
[Information request] Are the android app and the website open-source ?

### DISCUSSION THREAD

**Discussion by Djazouli** [birdnet-team/BirdNET-Analyzer#350] [Q&A]
Hello, first of all, I am deeply sorry, this is not a bug, but I could not find the answer somewhere else.

I wanted to know if the Android app/the website with the live submission map were by any chance open-sourced.
I would really like to see if I could do some performance improvements on the initial load of the live submission map, or fork the Android app to add some kind of "Pokedex" (I would love to see all the birds of my region, and which one I already encountered, and so on).

Is it something possible, or are both of these parts closed source ?

Thank you very much

**Comment by max-mauermann** [comment]
Hi,
Neither the Android app nor the submission map are open-source.

For the "pokedex" feature you might want to check the "explore your area" tab in the app (via the burger menu). It uses the geomodel to show which birds occur in your area at the current time of the year and should even show little checkmarks if you have recordings for them.

**Comment by max-mauermann** [comment]
Just a quick follow-up to that:

The map data can be requested from https://birdnet.cornell.edu/map/requeststats so you could build/host a map by yourself (we could then link to this). Also the map is just client-side rendering with html+javscript so you can get the code for it from the website to get started.

If you want to contribute to a birding app i would also like to mention [whoBIRD](https://github.com/woheller69/whoBIRD), which is an open-source birding app on the f-droid store that uses the birdnet model on device.
